index.html

```diff
@@ -254,7 +250,7 @@ <h3 class="project-title">Professional Art Printing Data</h3>
             </ul>
         </section>
         <section class="about-me decorative-background2">
-            <h2 class="h2-about-me">About me</h2>
+            <div class="h2-about-me">About me</div>
```

Comment:
[ Optional Changes ] 

- [ ] Kindly make this a heading text because this is meant to be the heading for the About Me section. For heading hierarchy purposes you can make it a heading 3 as you did with the other headings. 👍